patches/minecraft/net/minecraft/world/storage/loot/LootTableManager.java.patch

```diff
@@ -5,7 +5,7 @@
                      try
                      {
 -                        return (LootTable)LootTableManager.field_186526_b.fromJson(s, LootTable.class);
-+                        return net.minecraftforge.common.ForgeHooks.loadLootTable(LootTableManager.field_186526_b, p_186517_1_, s, true);
```

Comment:
After all, I think you should change the call to `ClassLoader.getResource` above, which prevents compatibility hacks.